recipe/meta.yaml

```
@@ -6,59 +6,79 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nutpie-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: f14282e2ac045c67a9b262a865b02a243178c55b541b236b21dfcb0c3678bcea
 
 build:
   number: 0
-  skip: true  # [py<310]
-  ignore_run_exports_from:
-    - libclang
+  # Pkg numba==0.59.1 is not avaliable on py313
+  skip: True  # [py<310 or py>312]
+  # clang and libclang does not exists on s390x
+  skip: True  # [s390x]
+  script_env:
+    - LIBCLANG_PATH=$BUILD_PREFIX/lib  # [not win]
+    - LIBCLANG_PATH=%LIBRARY_BIN%  # [win]
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - maturin >=1.1,<2.0                     # [build_platform != target_platform]
+    - python
     - {{ compiler('c') }}
-    - {{ stdlib("c") }}
     - {{ compiler('rust') }}
-    - libclang
-    - clang
     - cargo-bundle-licenses
+    - libclang  # [linux or osx]
+    - clang  # [linux or osx]
+    # Unable to find libclang: "couldn't find any valid shared libraries matching: ['clang.dll', 'libclang.dll'], set the `LIBCLANG_PATH` environment variable to a path where one of these files can be found (invalid: [])"
+    # LIBCLANG_PATH: %PREFIX%\Library\bin don`t have inside libclang
+    - clang  # [win]
```

If I read between the lines, here, `clang` is required on all platforms and the comment feels redundant given that it is repeated in `bld.bat` so I might go with just:

```suggestion
    - clang
```

---

done